js-loops/js-loops_forin-select-nav/js/index.js

```diff
@@ -21,7 +21,11 @@ select.name = "languages";
 main.append(select);
 
 // --v-- write/change code here --v--
-
+for (const key in languages) {
+  const optionElement = document.createElement("option");
+  optionElement.textContent = languages[key];
```

Comment:
Option must have a value attribute to <option value="EN">English</option>

Reply:
means that i have to do `optionElemen.setAttribute()` here?